en/docs/learn/api-controller/advanced-topics/configuring-environment-specific-parameters.md

```diff
@@ -59,7 +87,19 @@ The following code snippet contains sample configuration of the parameter file.
               production:
                   url: 'https://test.wso2.com'
                   config:
-                      retryTimeOut: $RETRY
+                    factor: 3
+                    suspendMaxDuration: 25000
+                    suspendDuration: 45000
+                    suspendErrorCode: 
+                        - "101504"
+                        - "101501"
+                    retryTimeOut: $RETRY
+                    retryDelay: 23000
+                    retryErroCode:
+                        - "101503" 
+                        - "101504"
+                    actionSelect: discard
+                    actionDuration: 75000
```

Comment:
```suggestion
                      factor: 3
                      suspendMaxDuration: 25000
                      suspendDuration: 45000
                      suspendErrorCode: 
                          - "101504"
                          - "101501"
                      retryTimeOut: $RETRY
                      retryDelay: 23000
                      retryErroCode:
                          - "101503" 
                          - "101504"
                      actionSelect: discard
                      actionDuration: 75000
```
```suggestion
                      factor: 3
```
Let's indent all the fields like this? (the earlier _retryTimeOut: $RETRY_ had correct indentation I guess)

Reply:
Fixed by [dca22dd](https://github.com/wso2/docs-apim/pull/5955/commits/dca22dd60544d2d61fe44191e9c167bcb16cafed)